extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/thing/ONE.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="sonos"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Player ONE Thing Type -->
+	<thing-type id="One" listed="false">
```

Comment:
Shouldn't this be better "ONE" in capitals (to be in sync with the PLAYXs?

Reply:
Well - it could be - but the the discovery has to be tuned too because the player announces itself as 'One' and not 'ONE'

Reply:
The discovery code could ignore the case.

Reply:
I'd agree that it would be better to harden the discovery code.

Reply:
The real Name of Play 1 is "PLAY:1" and the real name of the one is "One" and NOT "ONE". So i recommend to use the real name for the ID. Sonos changed their pattern: http://www.sonos.com/de-de/shop/one.html

Reply:
Ok, but then these lines are clearly wrong in the first place: https://github.com/eclipse/smarthome/pull/4429/files#diff-148f59c83d8c2a479dea48c22dd67b70R10